mkdocs.yml

```diff
@@ -193,6 +193,8 @@ nav:
       - 'Schedule Presets': 'presets-schedule.md'
       - 'Workaround Presets': 'presets-workarounds.md'
   - All Other:
+      - User Stories:
+          - 'Swissquote': 'user-stories/swissquote.md'
```

Comment:
We can start like this but longer term I don't think we should have each story in the sidebar

Reply:
Can you explain more? I don't understand why you think this. 😉

Reply:
Let's say we have 5-10 user stories. We don't want the sidebar to be too long. Or are third level items collapsed by default?

Reply:
The "User Stories" part is collapsed by default, you must click to expand it.